src/components/dataset/overview/dataset-properties.vue

```diff
@@ -95,7 +95,7 @@ export default {
 <i18n lang="json5">
   {
     "en": {
-      "emptyTable": "There are no Properties in this Dataset."
+      "emptyTable": "The Entities in this Entity List do not have any user-defined Properties."
```

Comment:
Just an idea, what do you think about "custom" properties instead of "user-defined"?

Reply:
I went with "user-defined" to match terminology we've used at https://getodk.github.io/xforms-spec/entities and elsewhere. I feel unease about custom but I'm not sure why. user-defined doesn't actually provide more info, does it? How about we leave it for now and see whether how we use the "user-defined properties" concept and whether we want to rename it across the board.